transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/specification/wse/connector/OpeningIT.java

```diff
@@ -125,31 +125,6 @@ public void shouldEstablishConnectionWithRequestHeaderXWebSocketProtocol() throw
         k3po.finish();
     }
 
```

Comment:
The problem is to set up a classloader (so that tests are isolated) and hence I moved them to separate tests.